### PR TITLE
create asset table and model

### PR DESCRIPTION
**Pull request by msm-faraj**

### CODE REVIEW

File: src/migrations/20240903125456-create-asset-table.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        type: Sequelize.UUID,
+        allowNull: false,
+        references: {
+          model: "portfolios",
+          key: "id",
+          name: "fk_assets_portfolios_id",
+        },
+      },
+      base_currency: {
+        type: Sequelize.ENUM(
```
Comment (pouyapourbaba) [inline src/migrations/20240903125456-create-asset-table.js:24]: There is a const for this, let's use that

File: src/migrations/20240903125456-create-asset-table.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          "NZD",
+          "KRW",
+          "SEK",
+          "NOK",
+          "TRY"
+        ),
+        allowNull: false,
+        defaultValue: "EUR",
+      },
+      remainig: {
```
Comment (pouyapourbaba) [inline src/migrations/20240903125456-create-asset-table.js:49]:
```suggestion
      quantity: {
```

File: src/migrations/20240903125456-create-asset-table.js (hunk 1 above)
Comment (msm-faraj) [reply src/migrations/20240903125456-create-asset-table.js:24]: but as much as I remember you told me not use it in migration and jsut use it in the model.

File: src/migrations/20240903125456-create-asset-table.js (hunk 2 above)
Comment (msm-faraj) [reply src/migrations/20240903125456-create-asset-table.js:49]: done

File: src/migrations/20240903125456-create-asset-table.js (hunk 1 above)
Comment (msm-faraj) [reply src/migrations/20240903125456-create-asset-table.js:24]: done for all

**Review by pouyapourbaba — APPROVED** [review]

**Pull request merged**